### PR TITLE
fixed binary API problem

### PR DESCRIPTION
**Pull request by jubecker**
This is a fix for https://github.com/lucidsoftware/neo-sbt-scalafmt/issues/69.

Please let me know what you think about the used versions.

### CODE REVIEW

**Comment by aparo** [comment]
This is blocking to upgrade to 2.12.6. May you merge and do a new release?

Thank you very much.

File: sbt-scalafmt/src/main/scala/com/lucidchart/sbt/scalafmt/ScalafmtCorePlugin.scala [hunk 1]
```diff
@@ -260,11 +260,11 @@ object ScalafmtCorePlugin extends AutoPlugin {
     libraryDependencies in Scalafmt := {
       val (scalaBinaryVersion, fmtVersion) = "(\\d+.){0,1}\\d+".r.findPrefixOf(scalafmtVersion.value) match {
         case Some("0.6")                               => ("2.11", "0.6")
-        case Some("0.7")                               => ("2.12", "1.0")
-        case Some(version) if version.startsWith("1.") => ("2.12", "1.0")
+        case Some("0.7")                               => ("2.12", "1.3")
+        case Some(version) if version.startsWith("1.") => ("2.12", "1.3")
         case _ =>
           println(s"Warning: Unknown Scalafmt version ${scalafmtVersion.value}; using 1.0 interface")
```
Comment (anwilkinson) [inline sbt-scalafmt/src/main/scala/com/lucidchart/sbt/scalafmt/ScalafmtCorePlugin.scala]: `using 1.0 interface` should probably also be updated to say 1.3.

**Comment by mrtnrdl** [comment]
Is there anything that is blocking this merge? Maybe I could help somewhere.

**Comment by jubecker** [comment]
Fixed the wrong warning message. Please let me know if anything else needs to be fixed.

**Comment by mwz** [comment]
👍

File: build.sbt [hunk 2]
```diff
@@ -71,7 +71,7 @@ lazy val sbtVersionAxis = new DefaultAxis {
         },
         scalaVersion := (sbtVersion.value match {
           case version if version.startsWith("0.13.") => "2.10.6"
-          case version if version.startsWith("1.0.")  => "2.12.2"
+          case version if version.startsWith("1.0.")  => "2.12.6"
```
Comment (gregghz) [inline build.sbt:74]: this should probably be changed to `if version.startsWith("1.")`

File: build.sbt [hunk 3]
```diff
@@ -44,8 +44,8 @@ lazy val `scalafmt-impl` = project
 lazy val `scalafmt-impl-0.6` = `scalafmt-impl`("0.6.8").settings(
   scalaVersion := "2.11.8"
 )
-lazy val `scalafmt-impl-1.0` = `scalafmt-impl`("1.0.0-RC3").settings(
-  scalaVersion := "2.12.2"
+lazy val `scalafmt-impl-1.3` = `scalafmt-impl`("1.3.0").settings(
```
Comment (gregghz) [inline build.sbt:47]: Does the `1.0.0-RC3` reference the sbt version or the scalafmt version?

**Pull request merged**